v2/index.html

```diff
@@ -1816,8 +1816,8 @@ <h3>Measure and Staff Symbols</h3>
             <h4>Bar Lines</h4>
             <p>
                 Bar lines serve to divide a musical staff into <a href="#measures">measures</a>.
-                Empty measures&mdash;two consecutive bar lines with no musical notation
-                between them&mdash;MUST NOT appear in the encoding.
+                Empty measures (two consecutive bar lines) are not permitted and MUST NOT be
```

Comment:
I think this is problematic because two consecutive bar lines are permitted (as a double bar line)

Reply:
They can't encode an empty measure with `//` since that MUST always be interpreted as a double bar line. This rule is to prevent `///` and `////`, etc.

Reply:
It also removes any ambiguity that `//` is two single bar lines and an empty measure. Since empty bars (measures) cannot appear, then there is no way to interpret that as anything other than a single (double) bar line.